src/libs/actions/Policy.ts

```diff
@@ -3565,6 +3565,64 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
 }
 
 function enablePolicyTaxes(policyID: string, enabled: boolean) {
+    const taxRatesData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: CONST.DEFAULT_TAX.RATES,
+                },
+            },
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: {
+                        taxes: {
+                            ...Object.keys(CONST.DEFAULT_TAX.RATES.taxes).reduce(
+                                (prevTaxesData, taxKey) => ({
+                                    ...prevTaxesData,
+                                    [taxKey]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD},
+                                }),
+                                {},
+                            ),
+                        },
+                    },
+                },
+            },
```

Comment:
Does this suggestion not work?
```suggestion
            {
                onyxMethod: Onyx.METHOD.MERGE,
                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
                value: {
                    taxRates: {
                        ...CONST.DEFAULT_TAX.RATES,
                        taxes: {
                            ...Object.keys(CONST.DEFAULT_TAX.RATES.taxes).reduce(
                                (prevTaxesData, taxKey) => ({
                                    ...prevTaxesData,
                                    [taxKey]: {pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD},
                                }),
                                {},
                            ),
                        },
                    },
                },
            },
```

Reply:
@luacmartins This can lead to typecheck error because of the CONST.DEFAULT_TAX.taxes is read-only. Do you want to apply your change and fix the typecheck?

Reply:
I'm not sure that I see how we're writing to `CONST.DEFAULT_TAX.taxes`

Reply:
In line 3698 in your suggestion code, we are writing to to "taxes" field

Reply:
Hmm I don't get any TS errors when I apply that suggestion

Reply:
did you try to run: npm run typecheck ?

Reply:
yes, no errors

Reply:
You can see this [commit](https://github.com/Expensify/App/pull/38375/commits/0cd290ef4a2a3f8a68630ffedbf703178ad61b07), that i tried to implement your suggestion code

Reply:
I think it is better to combine the two updates.
@nkdengineer 
Can you please check if the following works? 
`…((CONST.DEFAULT_TAX as TaxRatesWithDefault).taxes[taxKey])`

Reply:
Thanks @rojiphil. It works. I updated PR

Reply:
Oh! I am not sure if we would want to import `TaxRatesWithDefault` type in CONST. I think CONST is quite a generic one to import any context specific types there. 
@luacmartins What is your take on this?

Reply:
Yea, we shouldn't import any types there